docs/design/enforceHostNetwork.md

```diff
@@ -0,0 +1,28 @@
+# Enforcing host network design document
+
+By default, the ceph-csi controller plugin operates on the pod network
+but under some circumstances, like provider mode with a dedicated storage network, it is necessary 
+to run the Ceph-CSI pods on the host network.
```

Comment:
`Ceph-CSI pods` to `Ceph-CSI controller plugin pods`

Reply:
done